src/model/person.ts

```diff
@@ -33,6 +34,69 @@ export class Person extends BackmatterEntity {
     }
   }
 
+  public toXml(affiliations: Affiliation[] = []): Element {
+    const xmlDoc = new DOMImplementation().createDocument(null, null);
+    const contrib = xmlDoc.createElement('contrib');
+    contrib.setAttribute('contrib-type', 'author');
+    contrib.setAttribute('id', this.id);
+    if (this.isCorrespondingAuthor) {
+      contrib.setAttribute('corresp', 'yes');
+    }
+  
+    const name = xmlDoc.createElement('name');
+    contrib.appendChild(name);
+    if (this.firstName) {
+      const firstName = xmlDoc.createElement('given-names');
+      firstName.appendChild(xmlDoc.createTextNode(this.firstName));
+      name.appendChild(firstName);
+    }
+  
+    if (this.lastName) {
+      const lastName = xmlDoc.createElement('surname');
+      lastName.appendChild(xmlDoc.createTextNode(this.lastName));
+      name.appendChild(lastName);
+    }
+  
+    if (this.suffix) {
+      const suffix = xmlDoc.createElement('suffix');
+      suffix.appendChild(xmlDoc.createTextNode(this.suffix));
+      name.appendChild(suffix);
+    }
+  
+    if (this.orcid) {
+      const orcidEl = xmlDoc.createElement('contrib-id');
+      orcidEl.setAttribute('contrib-id-type', 'orcid');
+      orcidEl.setAttribute('authenticated', String(this.isAuthenticated));
+      orcidEl.appendChild(xmlDoc.createTextNode(`https://orcid.org/${this.orcid}`));
```

Comment:
Looking [here](https://github.com/libero/editor/issues/24), I think you only need the number rather than the URL.